### PR TITLE
Update client error to show link in console

### PR DESCRIPTION
**Pull request by ijjk**
This updates our default client-side error to show the developer info link in the console to help prevent users from confusing it with a debug link for the application itself.

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Build</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| buildDuration | 15.6s | 15s | -616ms |
| buildDurationCached | 3.5s | 3.4s | -69ms |
| nodeModulesSize | 50.2 MB | 50.2 MB | ⚠️ +14 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.602 | 2.59 | -0.01 |
| / avg req/sec | 960.83 | 965.39 | +4.56 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.456 | 1.44 | -0.02 |
| /error-in-render avg req/sec | 1717.59 | 1735.77 | +18.18 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| 745.HASH.js gzip | 179 B | 179 B | ✓ |
| framework-HASH.js gzip | 42.2 kB | 42.2 kB | ✓ |
| main-HASH.js gzip | 23.1 kB | 23.1 kB | ⚠️ +38 B |
| webpack-HASH.js gzip | 1.5 kB | 1.5 kB | ✓ |
| Overall change | 67 kB | 67 kB | ⚠️ +38 B |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| _app-HASH.js gzip | 980 B | 980 B | ✓ |
| _error-HASH.js gzip | 194 B | 194 B | ✓ |
| amp-HASH.js gzip | 312 B | 312 B | ✓ |
| css-HASH.js gzip | 329 B | 329 B | ✓ |
| dynamic-HASH.js gzip | 2.52 kB | 2.52 kB | ✓ |
| head-HASH.js gzip | 350 B | 350 B | ✓ |
| hooks-HASH.js gzip | 904 B | 904 B | ✓ |
| image-HASH.js gzip | 4.12 kB | 4.12 kB | ✓ |
| index-HASH.js gzip | 261 B | 261 B | ✓ |
| link-HASH.js gzip | 1.66 kB | 1.66 kB | ✓ |
| routerDirect..HASH.js gzip | 319 B | 319 B | ✓ |
| script-HASH.js gzip | 387 B | 387 B | ✓ |
| withRouter-HASH.js gzip | 320 B | 320 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 12.8 kB | 12.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| _buildManifest.js gzip | 490 B | 490 B | ✓ |
| Overall change | 490 B | 490 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| index.html gzip | 532 B | 531 B | -1 B |
| link.html gzip | 544 B | 543 B | -1 B |
| withRouter.html gzip | 524 B | 524 B | ✓ |
| Overall change | 1.6 kB | 1.6 kB | -2 B |

</details>

#### Diffs
<details>
<summary>Diff for <strong>main-HASH.js</strong></summary>

```diff
@@ -1528,6 +1528,9 @@
         } // Make sure we log the error to the console, otherwise users can't track down issues.
 
         console.error(err);
+        console.error(
+          "A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred"
+        );
         return pageLoader
           .loadPage("/_error")
           .then(function(_ref4) {
@@ -3599,16 +3602,7 @@
                         : /*#__PURE__*/ _react["default"].createElement(
                             _react["default"].Fragment,
                             null,
-                            "Application error: a client-side exception has occurred (",
-                            /*#__PURE__*/ _react["default"].createElement(
-                              "a",
-                              {
-                                href:
-                                  "https://nextjs.org/docs/messages/client-side-exception-occurred"
-                              },
-                              "developer guidance"
-                            ),
-                            ")"
+                            "Application error: a client-side exception has occurred (see the console for more information)"
                           ),
                       "."
                     )
```
</details>
<details>
<summary>Diff for <strong>index.html</strong></summary>

```diff
@@ -19,7 +19,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/main-269302b79c85b2c7608d.js"
+      src="/_next/static/chunks/main-1de4f0b4230f10ddd49a.js"
       defer=""
     ></script>
     <script
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -19,7 +19,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/main-269302b79c85b2c7608d.js"
+      src="/_next/static/chunks/main-1de4f0b4230f10ddd49a.js"
       defer=""
     ></script>
     <script
```
</details>
<details>
<summary>Diff for <strong>withRouter.html</strong></summary>

```diff
@@ -19,7 +19,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/main-269302b79c85b2c7608d.js"
+      src="/_next/static/chunks/main-1de4f0b4230f10ddd49a.js"
       defer=""
     ></script>
     <script
```
</details>
</details>
<hr/>
<details>
<summary><strong>Webpack 4 Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| buildDuration | 12.1s | 12.3s | ⚠️ +212ms |
| buildDurationCached | 4.7s | 4.8s | ⚠️ +149ms |
| nodeModulesSize | 50.2 MB | 50.2 MB | ⚠️ +14 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.584 | 2.626 | ⚠️ +0.04 |
| / avg req/sec | 967.37 | 951.96 | ⚠️ -15.41 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.431 | 1.473 | ⚠️ +0.04 |
| /error-in-render avg req/sec | 1747.32 | 1697.29 | ⚠️ -50.03 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| 17.HASH.js gzip | 185 B | 185 B | ✓ |
| 677f882d2ed8..HASH.js gzip | 14 kB | 14 kB | ✓ |
| framework.HASH.js gzip | 41.9 kB | 41.9 kB | ✓ |
| main-HASH.js gzip | 10.5 kB | 10.6 kB | ⚠️ +13 B |
| webpack-HASH.js gzip | 1.19 kB | 1.19 kB | ✓ |
| Overall change | 67.8 kB | 67.8 kB | ⚠️ +13 B |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| _app-HASH.js gzip | 965 B | 965 B | ✓ |
| _error-HASH.js gzip | 3.74 kB | 3.71 kB | -32 B |
| amp-HASH.js gzip | 552 B | 552 B | ✓ |
| css-HASH.js gzip | 333 B | 333 B | ✓ |
| dynamic-HASH.js gzip | 2.71 kB | 2.71 kB | ✓ |
| head-HASH.js gzip | 2.97 kB | 2.97 kB | ✓ |
| hooks-HASH.js gzip | 911 B | 911 B | ✓ |
| index-HASH.js gzip | 231 B | 231 B | ✓ |
| link-HASH.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..HASH.js gzip | 298 B | 298 B | ✓ |
| script-HASH.js gzip | 2.94 kB | 2.94 kB | ✓ |
| withRouter-HASH.js gzip | 294 B | 294 B | ✓ |
| e025d2764813..52f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 17.7 kB | 17.7 kB | -32 B |

</details>

<details>
<summary><strong>Client Build Manifests</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| _buildManifest.js gzip | 499 B | 498 B | -1 B |
| Overall change | 499 B | 498 B | -1 B |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| index.html gzip | 577 B | 578 B | ⚠️ +1 B |
| link.html gzip | 589 B | 590 B | ⚠️ +1 B |
| withRouter.html gzip | 570 B | 571 B | ⚠️ +1 B |
| Overall change | 1.74 kB | 1.74 kB | ⚠️ +3 B |

</details>

#### Diffs
<details>
<summary>Diff for <strong>_buildManifest.js</strong></summary>

```diff
@@ -2,7 +2,7 @@ self.__BUILD_MANIFEST = {
   __rewrites: { beforeFiles: [], afterFiles: [], fallback: [] },
   "/": ["static\u002Fchunks\u002Fpages\u002Findex-b8f868a0fff56c820ae0.js"],
   "/_error": [
-    "static\u002Fchunks\u002Fpages\u002F_error-37497c6ae37781198b55.js"
+    "static\u002Fchunks\u002Fpages\u002F_error-20ca739d18aefd3169be.js"
   ],
   "/amp": ["static\u002Fchunks\u002Fpages\u002Famp-4855291bfb1c5a551232.js"],
   "/css": [
```
</details>
<details>
<summary>Diff for <strong>_error-HASH.js</strong></summary>

```diff
@@ -145,16 +145,7 @@ _N_E = (window["webpackJsonp_N_E"] = window["webpackJsonp_N_E"] || []).push([
                         : /*#__PURE__*/ _react["default"].createElement(
                             _react["default"].Fragment,
                             null,
-                            "Application error: a client-side exception has occurred (",
-                            /*#__PURE__*/ _react["default"].createElement(
-                              "a",
-                              {
-                                href:
-                                  "https://nextjs.org/docs/messages/client-side-exception-occurred"
-                              },
-                              "developer guidance"
-                            ),
-                            ")"
+                            "Application error: a client-side exception has occurred (see the console for more information)"
                           ),
                       "."
                     )
```
</details>
<details>
<summary>Diff for <strong>main-HASH.js</strong></summary>

```diff
@@ -145,16 +145,7 @@ _N_E = (window["webpackJsonp_N_E"] = window["webpackJsonp_N_E"] || []).push([
                         : /*#__PURE__*/ _react["default"].createElement(
                             _react["default"].Fragment,
                             null,
-                            "Application error: a client-side exception has occurred (",
-                            /*#__PURE__*/ _react["default"].createElement(
-                              "a",
-                              {
-                                href:
-                                  "https://nextjs.org/docs/messages/client-side-exception-occurred"
-                              },
-                              "developer guidance"
-                            ),
-                            ")"
+                            "Application error: a client-side exception has occurred (see the console for more information)"
                           ),
                       "."
                     )
@@ -2073,6 +2064,9 @@ _N_E = (window["webpackJsonp_N_E"] = window["webpackJsonp_N_E"] || []).push([
         } // Make sure we log the error to the console, otherwise users can't track down issues.
 
         console.error(err);
+        console.error(
+          "A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred"
+        );
         return pageLoader
           .loadPage("/_error")
           .then(function(_ref4) {
```
</details>
<details>
<summary>Diff for <strong>index.html</strong></summary>

```diff
@@ -23,7 +23,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/main-bc84b4c21c976d449f85.js"
+      src="/_next/static/chunks/main-0ca3a304590870191b8b.js"
       defer=""
     ></script>
     <script
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -23,7 +23,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/main-bc84b4c21c976d449f85.js"
+      src="/_next/static/chunks/main-0ca3a304590870191b8b.js"
       defer=""
     ></script>
     <script
```
</details>
<details>
<summary>Diff for <strong>withRouter.html</strong></summary>

```diff
@@ -23,7 +23,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/main-bc84b4c21c976d449f85.js"
+      src="/_next/static/chunks/main-0ca3a304590870191b8b.js"
       defer=""
     ></script>
     <script
```
</details>
</details>
Commit: 30ac0cdd29fe275d9f8b70895dd0e9e3db37e4e1

File: packages/next/pages/_error.tsx [hunk 1]
```diff
@@ -57,11 +57,8 @@ export default class Error<P = {}> extends React.Component<P & ErrorProps> {
                 title
               ) : (
                 <>
-                  Application error: a client-side exception has occurred (
-                  <a href="https://nextjs.org/docs/messages/client-side-exception-occurred">
-                    developer guidance
-                  </a>
-                  )
+                  Application error: a client-side exception has occurred (see
```
Comment (styfle) [inline packages/next/pages/_error.tsx:60]: Is this browser dev console or the terminal console running next cli?

File: test/integration/no-page-props/test/index.test.js [hunk 2]
```diff
@@ -74,7 +74,7 @@ const runTests = () => {
     await browser.waitForElementByCss('h2')
     expect(await browser.eval('window.beforeNav')).toBe(null)
     expect(await browser.elementByCss('h2').text()).toBe(
-      'Application error: a client-side exception has occurred (developer guidance).'
+      'Application error: a client-side exception has occurred (see the console for more information).'
```
Comment (ijjk) [inline test/integration/no-page-props/test/index.test.js:77]:
```suggestion
      'Application error: a client-side exception has occurred (see the browser console for more information).'
```

File: test/integration/no-page-props/test/index.test.js [hunk 3]
```diff
@@ -39,7 +39,7 @@ const runTests = () => {
   it('should load 404 page correctly', async () => {
     const browser = await webdriver(appPort, '/non-existent')
     expect(await browser.elementByCss('h2').text()).toBe(
-      'Application error: a client-side exception has occurred (developer guidance).'
+      'Application error: a client-side exception has occurred (see the console for more information).'
```
Comment (ijjk) [inline test/integration/no-page-props/test/index.test.js:42]:
```suggestion
      'Application error: a client-side exception has occurred (see the browser console for more information).'
```

File: test/integration/css/test/index.test.js [hunk 4]
```diff
@@ -1509,7 +1509,7 @@ describe('CSS Support', () => {
           await browser.waitForElementByCss('#link-other').click()
           await check(
             () => browser.eval(`document.body.innerText`),
-            'Application error: a client-side exception has occurred (developer guidance).',
+            'Application error: a client-side exception has occurred (see the console for more information).',
```
Comment (ijjk) [inline test/integration/css/test/index.test.js:1512]:
```suggestion
            'Application error: a client-side exception has occurred (see the browser console for more information).',
```

File: packages/next/pages/_error.tsx [hunk 5]
```diff
@@ -57,11 +57,8 @@ export default class Error<P = {}> extends React.Component<P & ErrorProps> {
...
+                  the console for more information)
```
Comment (ijjk) [inline packages/next/pages/_error.tsx:61]:
```suggestion
                  the browser console for more information)
```

File: packages/next/pages/_error.tsx (hunk 1 above)
Comment (ijjk) [reply packages/next/pages/_error.tsx:60]: Updated to stress it's the browser console

**Review by styfle — APPROVED** [review]
Thanks!

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Build</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| buildDuration | 15.9s | 15.9s | ⚠️ +5ms |
| buildDurationCached | 3.6s | 3.6s | -54ms |
| nodeModulesSize | 50.2 MB | 50.2 MB | ⚠️ +28 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.713 | 2.703 | -0.01 |
| / avg req/sec | 921.48 | 924.83 | +3.35 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.54 | 1.491 | -0.05 |
| /error-in-render avg req/sec | 1623.58 | 1676.48 | +52.9 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| 745.HASH.js gzip | 179 B | 179 B | ✓ |
| framework-HASH.js gzip | 42.2 kB | 42.2 kB | ✓ |
| main-HASH.js gzip | 23.1 kB | 23.1 kB | ⚠️ +49 B |
| webpack-HASH.js gzip | 1.5 kB | 1.5 kB | ✓ |
| Overall change | 67 kB | 67 kB | ⚠️ +49 B |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| _app-HASH.js gzip | 980 B | 980 B | ✓ |
| _error-HASH.js gzip | 194 B | 194 B | ✓ |
| amp-HASH.js gzip | 312 B | 312 B | ✓ |
| css-HASH.js gzip | 329 B | 329 B | ✓ |
| dynamic-HASH.js gzip | 2.52 kB | 2.52 kB | ✓ |
| head-HASH.js gzip | 350 B | 350 B | ✓ |
| hooks-HASH.js gzip | 904 B | 904 B | ✓ |
| image-HASH.js gzip | 4.12 kB | 4.12 kB | ✓ |
| index-HASH.js gzip | 261 B | 261 B | ✓ |
| link-HASH.js gzip | 1.66 kB | 1.66 kB | ✓ |
| routerDirect..HASH.js gzip | 319 B | 319 B | ✓ |
| script-HASH.js gzip | 387 B | 387 B | ✓ |
| withRouter-HASH.js gzip | 320 B | 320 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 12.8 kB | 12.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| _buildManifest.js gzip | 490 B | 490 B | ✓ |
| Overall change | 490 B | 490 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| index.html gzip | 531 B | 531 B | ✓ |
| link.html gzip | 544 B | 542 B | -2 B |
| withRouter.html gzip | 524 B | 523 B | -1 B |
| Overall change | 1.6 kB | 1.6 kB | -3 B |

</details>

#### Diffs
<details>
<summary>Diff for <strong>main-HASH.js</strong></summary>

```diff
@@ -1528,6 +1528,9 @@
         } // Make sure we log the error to the console, otherwise users can't track down issues.
 
         console.error(err);
+        console.error(
+          "A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred"
+        );
         return pageLoader
           .loadPage("/_error")
           .then(function(_ref4) {
@@ -3599,16 +3602,7 @@
                         : /*#__PURE__*/ _react["default"].createElement(
                             _react["default"].Fragment,
                             null,
-                            "Application error: a client-side exception has occurred (",
-                            /*#__PURE__*/ _react["default"].createElement(
-                              "a",
-                              {
-                                href:
-                                  "https://nextjs.org/docs/messages/client-side-exception-occurred"
-                              },
-                              "developer guidance"
-                            ),
-                            ")"
+                            "Application error: a client-side exception has occurred (see the browser console for more information)"
                           ),
                       "."
                     )
```
</details>
<details>
<summary>Diff for <strong>index.html</strong></summary>

```diff
@@ -19,7 +19,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/main-bb71ff97260681ef94d1.js"
+      src="/_next/static/chunks/main-9b944557fb55cbe9ddb3.js"
       defer=""
     ></script>
     <script
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -19,7 +19,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/main-bb71ff97260681ef94d1.js"
+      src="/_next/static/chunks/main-9b944557fb55cbe9ddb3.js"
       defer=""
     ></script>
     <script
```
</details>
<details>
<summary>Diff for <strong>withRouter.html</strong></summary>

```diff
@@ -19,7 +19,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/main-bb71ff97260681ef94d1.js"
+      src="/_next/static/chunks/main-9b944557fb55cbe9ddb3.js"
       defer=""
     ></script>
     <script
```
</details>
</details>
<hr/>
<details>
<summary><strong>Webpack 4 Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| buildDuration | 12.6s | 12.9s | ⚠️ +238ms |
| buildDurationCached | 5s | 4.9s | -93ms |
| nodeModulesSize | 50.2 MB | 50.2 MB | ⚠️ +28 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.686 | 2.698 | ⚠️ +0.01 |
| / avg req/sec | 930.76 | 926.47 | ⚠️ -4.29 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.503 | 1.521 | ⚠️ +0.02 |
| /error-in-render avg req/sec | 1662.86 | 1643.34 | ⚠️ -19.52 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| 17.HASH.js gzip | 185 B | 185 B | ✓ |
| 677f882d2ed8..HASH.js gzip | 14 kB | 14 kB | ✓ |
| framework.HASH.js gzip | 41.9 kB | 41.9 kB | ✓ |
| main-HASH.js gzip | 10.5 kB | 10.6 kB | ⚠️ +18 B |
| webpack-HASH.js gzip | 1.19 kB | 1.19 kB | ✓ |
| Overall change | 67.8 kB | 67.8 kB | ⚠️ +18 B |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| _app-HASH.js gzip | 965 B | 965 B | ✓ |
| _error-HASH.js gzip | 3.74 kB | 3.71 kB | -27 B |
| amp-HASH.js gzip | 552 B | 552 B | ✓ |
| css-HASH.js gzip | 333 B | 333 B | ✓ |
| dynamic-HASH.js gzip | 2.71 kB | 2.71 kB | ✓ |
| head-HASH.js gzip | 2.97 kB | 2.97 kB | ✓ |
| hooks-HASH.js gzip | 911 B | 911 B | ✓ |
| index-HASH.js gzip | 231 B | 231 B | ✓ |
| link-HASH.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..HASH.js gzip | 298 B | 298 B | ✓ |
| script-HASH.js gzip | 2.94 kB | 2.94 kB | ✓ |
| withRouter-HASH.js gzip | 294 B | 294 B | ✓ |
| e025d2764813..52f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 17.7 kB | 17.7 kB | -27 B |

</details>

<details>
<summary><strong>Client Build Manifests</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| _buildManifest.js gzip | 499 B | 498 B | -1 B |
| Overall change | 499 B | 498 B | -1 B |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/client-error | Change |
| - | - | - | - |
| index.html gzip | 577 B | 576 B | -1 B |
| link.html gzip | 589 B | 589 B | ✓ |
| withRouter.html gzip | 569 B | 570 B | ⚠️ +1 B |
| Overall change | 1.74 kB | 1.74 kB | ✓ |

</details>

#### Diffs
<details>
<summary>Diff for <strong>_buildManifest.js</strong></summary>

```diff
@@ -2,7 +2,7 @@ self.__BUILD_MANIFEST = {
   __rewrites: { beforeFiles: [], afterFiles: [], fallback: [] },
   "/": ["static\u002Fchunks\u002Fpages\u002Findex-b8f868a0fff56c820ae0.js"],
   "/_error": [
-    "static\u002Fchunks\u002Fpages\u002F_error-37497c6ae37781198b55.js"
+    "static\u002Fchunks\u002Fpages\u002F_error-76a050cae8d92043220a.js"
   ],
   "/amp": ["static\u002Fchunks\u002Fpages\u002Famp-4855291bfb1c5a551232.js"],
   "/css": [
```
</details>
<details>
<summary>Diff for <strong>_error-HASH.js</strong></summary>

```diff
@@ -145,16 +145,7 @@ _N_E = (window["webpackJsonp_N_E"] = window["webpackJsonp_N_E"] || []).push([
                         : /*#__PURE__*/ _react["default"].createElement(
                             _react["default"].Fragment,
                             null,
-                            "Application error: a client-side exception has occurred (",
-                            /*#__PURE__*/ _react["default"].createElement(
-                              "a",
-                              {
-                                href:
-                                  "https://nextjs.org/docs/messages/client-side-exception-occurred"
-                              },
-                              "developer guidance"
-                            ),
-                            ")"
+                            "Application error: a client-side exception has occurred (see the browser console for more information)"
                           ),
                       "."
                     )
```
</details>
<details>
<summary>Diff for <strong>main-HASH.js</strong></summary>

```diff
@@ -145,16 +145,7 @@ _N_E = (window["webpackJsonp_N_E"] = window["webpackJsonp_N_E"] || []).push([
                         : /*#__PURE__*/ _react["default"].createElement(
                             _react["default"].Fragment,
                             null,
-                            "Application error: a client-side exception has occurred (",
-                            /*#__PURE__*/ _react["default"].createElement(
-                              "a",
-                              {
-                                href:
-                                  "https://nextjs.org/docs/messages/client-side-exception-occurred"
-                              },
-                              "developer guidance"
-                            ),
-                            ")"
+                            "Application error: a client-side exception has occurred (see the browser console for more information)"
                           ),
                       "."
                     )
@@ -2073,6 +2064,9 @@ _N_E = (window["webpackJsonp_N_E"] = window["webpackJsonp_N_E"] || []).push([
         } // Make sure we log the error to the console, otherwise users can't track down issues.
 
         console.error(err);
+        console.error(
+          "A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred"
+        );
         return pageLoader
           .loadPage("/_error")
           .then(function(_ref4) {
```
</details>
<details>
<summary>Diff for <strong>index.html</strong></summary>

```diff
@@ -23,7 +23,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/main-bc84b4c21c976d449f85.js"
+      src="/_next/static/chunks/main-a44a2d9aee7189b3b90f.js"
       defer=""
     ></script>
     <script
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -23,7 +23,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/main-bc84b4c21c976d449f85.js"
+      src="/_next/static/chunks/main-a44a2d9aee7189b3b90f.js"
       defer=""
     ></script>
     <script
```
</details>
<details>
<summary>Diff for <strong>withRouter.html</strong></summary>

```diff
@@ -23,7 +23,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/main-bc84b4c21c976d449f85.js"
+      src="/_next/static/chunks/main-a44a2d9aee7189b3b90f.js"
       defer=""
     ></script>
     <script
```
</details>
</details>
Commit: b4deed14e3a6952afbe241515af4b8aa9d4d41cb

**Pull request merged**